repositories/ConsumerRepository.ts

```diff
@@ -19,7 +19,7 @@ const addFilterToQueryIfApplicable = (
   oktaQuery: OktaExpressions,
   filterType: FilterType = FilterType.CONTAINS,
 ): void => {
-  if (!filter || filter.length === 0) {
+  if (filter.length === 0) {
```

Comment:
will `filter.length === 0` if `filter` is undefined or will it throw an error that `filter` is undefined?

Reply:
The two spots this function is used (further down on L77 & L78) the default filter array being passed in is an empty array, so there shouldn't be an `undefined` situation.